lib/system/index.js

```diff
@@ -24,6 +25,7 @@ var capitalize = function(str) {
 module.exports    = system;
 system.os_name    = os_name;
 system.os_release = os.release();
+system.user_agent = 'Prey/' + version + ' (Node '+ process.version + ' ' + system.os_name[0].toUpperCase() + system.os_name.slice(1) + ' ' + system.os_release + ')';
```

Comment:
`os_name` comes from `process.platform`, which according to [the docs](https://nodejs.org/docs/v0.12.7/api/process.html#process_process_platform) returns strictly `darwin`, `freebsd`, `linux`, `sunos` or `win32`, otherwise `undefined` would be stated just like in [process.mainModule](https://nodejs.org/docs/v0.12.7/api/process.html#process_process_mainmodule).